portfolio/src/main/webapp/script.js

```diff
@@ -67,8 +70,9 @@ function fetchComments() {
       
       userInfo.append(picture);
       userInfo.appendChild(user);
-      if (gapi.auth2.getAuthInstance().isSignedIn.get()) {
-        const commenterEmail = gapi.auth2.init().currentUser.get().getBasicProfile().getEmail();
+      let auth2 = gapi.auth2.init(); 
+      if (auth2.isSignedIn) {
+        const commenterEmail = auth2.currentUser.le.Tt.Du;
```

Comment:
One thing to check is whether all the nested properties are always defined when the user is signed in. For example, this could throw errors if auth2.currentUser, auth2.currentUser.le, or auth2.currentUser.le.Tt is null or undefined.

If not sure, to be extra safe you can add explicit checks in the if above:
```
if (auth2.isSignedIn && auth2.currentUser && auth2.currentUser.le && auth2.currentUser.le.Tt) {
```